### ISSUE TITLE
XML editor Beautify creates 500 server response in production

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-online-tools#277]
To reproduce, upload an XML file and in the SPDX online tool XML edit function and select Beautify.

No further information could be found in the client log or in the server logs.

**Comment by goneall** [comment]
Found the following in the src/errors.log file:

```
ERROR : 2020-12-29 03:07:29,775 : Traceback (most recent call last):
  File "/spdx/src/app/views.py", line 1453, in beautify
    f.write(xmlString)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xa9' in position 413: ordinal not in range(128)
```

**Comment by rtgdk** [comment]
I think this was fixed with #283 . Closing this.
